### PR TITLE
Refactored the plugin naming and image

### PR DESCRIPTION
**Pull request by Vatyx**

### CODE REVIEW

File: SpatialGDK/Documentation/content/toolbars.md [hunk 1]
```diff
@@ -86,5 +86,5 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal
 
 | Setting | Description |
 | --- | --- |
-| Generate schema for all supported classes | **Experimental** Use this to generate [schema]({{urlRoot}}/content/glossary#schema) for all UObjects that have replicated or handover properties. As the GDK does not currently support multiple replicated Actor components of the same type on an Actor, classes extended from `USceneComponent` don't generate schemas. |
+| Generate schema for all supported classes | Use this to generate [schema]({{urlRoot}}/content/glossary#schema) for all network supported UObjects. Classes extended from `USceneComponent` don't generate schemas. **Highly recommended leave this option turned on.** |
```
Comment (m-samiec) [inline SpatialGDK/Documentation/content/toolbars.md]:
```suggestion
| Generate schema for all supported classes | Use this to generate [schema]({{urlRoot}}/content/glossary#schema) for all network supported UObjects. Classes extended from `USceneComponent` don't generate schemas. **It is highly recommended to leave this option turned on.** |
```

File: SpatialGDK/SpatialGDK.uplugin [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
-		{
-			"Name": "OnlineSubsystemUtils",
-			"Enabled": true
-		}
-	]
+  "FileVersion": 3,
+  "Version": 1,
+  "VersionName": "1.0",
+  "FriendlyName": "SpatialOS GDK for Unreal",
+  "Description": "The SpatialOS game development kit (GDK) for Unreal Engine allows you to host your game and combine multiple dedicated server instances across one seamless game world whilst using the Unreal Engine networking API.",
```
Comment (m-samiec) [inline SpatialGDK/SpatialGDK.uplugin]:
```suggestion
  "Description": "The SpatialOS Game Development Kit (GDK) for Unreal Engine allows you to host your game and combine multiple dedicated server instances across one seamless game world whilst using the Unreal Engine networking API.",
```

File: SpatialGDK/SpatialGDK.uplugin [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
-			"WhitelistPlatforms": [ "Win64", "Linux", "XboxOne", "PS4" ]
-		},
-	],
-	"Plugins": [
-		{
-			"Name": "OnlineSubsystemUtils",
-			"Enabled": true
-		}
-	]
+  "FileVersion": 3,
```
Comment (m-samiec) [inline SpatialGDK/SpatialGDK.uplugin]: Tabbing in this file has been removed.

File: SpatialGDKEditorToolbar/SpatialGDKEditorToolbar.uplugin [hunk 4]
```diff
@@ -1,30 +1,31 @@
...
-		{
-			"Name": "SpatialGDK",
-			"Enabled": true
-		}
-	]
+  "FileVersion": 3,
+  "Version": 1,
+  "VersionName": "1.0",
+  "FriendlyName": "SpatialOS GDK for Unreal Toolbar",
+  "Description": "The SpatialOS game development kit (GDK) for Unreal Engine allows you to host your game and combine multiple dedicated server instances across one seamless game world whilst using the Unreal Engine networking API.",
```
Comment (m-samiec) [inline SpatialGDKEditorToolbar/SpatialGDKEditorToolbar.uplugin]:
```suggestion
  "Description": "The SpatialOS Game Development Kit (GDK) for Unreal Engine allows you to host your game and combine multiple dedicated server instances across one seamless game world whilst using the Unreal Engine networking API.",
```

File: SpatialGDK/SpatialGDK.uplugin (hunk 3 above)
Comment (Vatyx) [reply SpatialGDK/SpatialGDK.uplugin]: I saw, that's ok. The files look the same now.

**Pull request merged**